package/src/components/Message/Message.tsx

```diff
@@ -751,8 +753,7 @@ const areEqual = <StreamChatGenerics extends DefaultStreamChatGenerics = Default
     prevMessage.status === nextMessage.status &&
     prevMessage.type === nextMessage.type &&
     prevMessage.text === nextMessage.text &&
-    prevMessage.pinned === nextMessage.pinned &&
-    prevMessage.updated_at === nextMessage.updated_at;
```

Comment:
What is the idea behind removing the updated_at here?

Reply:
@vishalnarkhede may have a better idea.